dune

```diff
@@ -25,11 +114,18 @@
  (deps (:source stdcompat__native.ml_native))
  (action (run cp %{source} %{targets})))
```

Comment:
Same here, this could use `copy_file`

Reply:
Yes. Also used that occasion to simplify them to use just `target` and `deps`.